src/reference-manual/pedantic-mode.Rmd

```diff
@@ -0,0 +1,363 @@
+
+
+# Pedantic Mode
+
+Pedantic Mode is a compilation option built into Stanc3 that warns you about potential issues in your Stan program.
```

Comment:
"Pedantic Mode" should not be capitalized.  More generally, technical terms and names of techniques should be lower case, like "no U-turn sampler" and "slice sampling" and so on.  Things like "Hamiltonian Monte Carlo" are not capitalized, they're just made up out of proper names.  Compare "Markov chain Monte Carlo" where the "chain" is lower case.